src/main/java/seedu/jxmusic/logic/parser/LibraryParser.java

```diff
@@ -15,6 +15,7 @@
 import seedu.jxmusic.logic.commands.ListCommand;
 import seedu.jxmusic.logic.commands.PauseCommand;
 import seedu.jxmusic.logic.commands.PlayPlaylistCommand;
+import seedu.jxmusic.logic.commands.PlaylistListCommand;
```

Comment:
I realised what a weird naming convention we have (line 17, 18)

Reply:
Feel free to suggest a better command phrase 😉